### DISCUSSION TITLE
How to apply Overriding ?

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [black7375/Firefox-UI-Fix#805] [Q&A]
[Overriding](https://github.com/black7375/Firefox-UI-Fix/blob/dev/docs/Project_Structure.md#overriding)

The steps that i followed ->

- Created `userChrome-overrides.css` in `~/.mozilla/firefox/6wdlrxc7.default-release/chrome`
- Content of `userChrome-overrides.css` are ->
```

.editBMPanel_locationRow {
  display: initial !important;
}
```
Which i copied from `userChrome.css` to test overriding.
- execute `./install.sh`
- I chose option `4) Update`
- It then asked `Select custom method` i chose `1) Append - Maintain changes in existing files and apply custom`

Outcome -> No effect

How to actually use [Overriding](https://github.com/black7375/Firefox-UI-Fix/blob/dev/docs/Project_Structure.md#overriding) section.

**Comment by black7375** [comment]
As far as I know, it has to work.
Run this code should be applied.

`bash -c "$(curl -fsSL https://raw.githubusercontent.com/black7375/Firefox-UI-Fix/master/install.sh)"`

---

I'll test it on the weekend.
Can you tell me what OS it is?

**Reply by KAGEYAM4** [reply]
Its working now, i uninstalled and reinstalled everything.



Also if i create a branch and keep my changes in it ( eg - `userChrome-overrides.css` `userContent-overrides.css` `user-overrides.js`), and if my branch is checked out. Will there be any problem when running `./install.sh`?
